kivakit-filesystems/java/src/main/java/project.properties

```diff
@@ -0,0 +1,4 @@
+project-name = KivaKit Extensions - Filesystems - JAVA
```

Comment:
Java -> Java

Reply:
what does it mean 'inline'?

Reply:
Oh, sorry. IntelliJ has a refactor called "inline variable"

var x = 9;
var y = x;

"inlined":

var y = 9;

You can do this is x isn't used anywhere else. Make sense?